es-ds-components/components/es-modal.vue

```diff
@@ -0,0 +1,107 @@
+<script setup lang="ts">
+import PrimeDialog from "primevue/dialog";
+
+const props = defineProps({
```

Comment:
I notice that the `scrollable` prop is no longer here. I'm not sure what this one does in the old version- could you look into that and verify whether we should perhaps continue to support it?

(Side note: It's definitely fine to leave out `id` as an explicit prop since it's no longer required for PrimeVue and will be passed through if provided anyways)